src/projects/project-create-pull-diff.html

```diff
@@ -0,0 +1,52 @@
+<link rel="import" href="../bower_components/neon-animation/neon-animation-runner-behavior.html">
+<link rel="import" href="../bower_components/neon-animation/animations/fade-in-animation.html">
+<link rel="import" href="../pull-request/pull-request-diff.html">
+<dom-module id="project-create-pull-diff">
+  <template>
+    <style>
+      :host {
+        display: none;
+        background-color: var(--primary-background-color);
+        padding: 2em;
+      }
+      h2 {
+        margin-top: 0;
+      }
+    </style>
+    <h2>Describe your changes</h2>
+    <pull-request-diff editable data-route="diff" diff="[[diff]]"></pull-request-diff>
```

Comment:
This should not be editable right? Only on the actual pull requests pages we want to make them editable?